walkthroughs/eks-app-mesh-multi-account/readme.md

```diff
@@ -0,0 +1,160 @@
+## Configure Frontend and Backend Account Profiles
+
+```
+cat ~/.aws/credentials
+
+[default]
+aws_access_key_id =  ...
+aws_secret_access_key = ...
+[frontend]
+aws_access_key_id =  ...
+aws_secret_access_key = ...
+[backend]
+aws_access_key_id = ...
+aws_secret_access_key = ...
+
+cat ~/.aws/config
+
+[default]
+region = sa-east-1
+[profile frontend]
+region = us-west-2
+[profile backend]
+region = us-west-2
+```
+
+## Deploy the Infrastructure
+
+```
+./infrasctucture/setup.sh
+```
+
+## Deploy EKS
+
+```
+./eks/setup.sh
+```
+
+## Deploy the App Mesh Controller on our Frontend Cluster
+
+```
+kubectl config use-context <iam_user>@am-multi-account-1.<region>.eksctl.io
+```
+
+```
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+```
+kubectl create ns appmesh-system
+helm upgrade -i appmesh-controller eks/appmesh-controller \
+--namespace appmesh-system
+
+kubectl -n appmesh-system get pods
+```
+
+## Deploy and Share Mesh
+
+```
+kubectl create ns yelb
```

Comment:
We can have a sample namespace spec in yaml format and apply it, we do that for all our other walkthroughs.